### PR TITLE
Merge hw1 into main

### PR DESCRIPTION
**Pull request by VladyslavPolushyn**
First attempt at release.

### CODE REVIEW

File: hw1/assets/sass/main.sass [hunk 1]
```diff
@@ -0,0 +1,1050 @@
...
+				&::before
+					content: ''
+					position: absolute
+					left: 0
+					width: 30px
+					height: 30px
+					background: url('../img/i-icon.png') center/cover
+
+	.glossary
+
```
Comment (marynakorol) [inline hw1/assets/sass/main.sass:419]: indent doesn't match the design 
image

**Review by marynakorol — CHANGES_REQUESTED** [review]

File: hw1/assets/js/accordion.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+const tipsAcc = [...document.querySelectorAll(".tips__accordion .accordion")];
+const tipsAccContent = [...document.querySelectorAll('.tips__accordion .accordion__content')];
+
+const faqsAcc = [...document.querySelectorAll(".faqs__accordion .accordion")];
+const faqsAccContent = [...document.querySelectorAll('.faqs__accordion .accordion__content')];
+
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/js/accordion.js:6]: Thank's. Done!

File: hw1/assets/js/slider.js [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+function removeActiveClass(arr) {
+
+	arr.forEach(item => {
+		item.classList.remove('active');
+	});
+
+}
+
+function renderSlides() {
+	if (window.matchMedia("(min-width: 767px)").matches) {
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/js/slider.js]: Done.

File: hw1/assets/js/slider.js [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+		    		<li>Software</li>
+		    		<li>SOC</li>
+		    		<li>Trojan Horse</li>
+		    		<li>VPN</li>
+		    		<li>Virus</li>
+		    		<li>Worm</li>
+		    	</ul>
+		    </div>
+		  </div>
+		`);
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/js/slider.js:138]: Thank's!
I will try to fix it.

File: hw1/assets/js/stickyNav.js [hunk 5]
```diff
@@ -0,0 +1,49 @@
+const tabLinks = document.querySelectorAll('.hero__tabs a');
+const tabNav = document.querySelector('.hero__tabs');
+const stickyTab = tabNav.offsetTop;
+
+const headerNav = document.querySelector('.main-header');
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/js/stickyNav.js]: Done.

File: hw1/assets/sass/main.sass [hunk 6]
```diff
@@ -0,0 +1,1050 @@
...
+					letter-spacing: 2px
+					text-transform: uppercase
+
+			.phone-number
+				font-size: 16px
+				font-style: normal
+				font-weight: 400
+				line-height: 36px
+
+			h5
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/sass/main.sass]: Thank's. Done!

File: hw1/index.html [hunk 7]
```diff
@@ -0,0 +1,325 @@
...
+					</nav>
+					<p class="main-phone">
+						Call the SOC <br>
+						<span class="phone-number">1-855-838-4500</span>
+					</p>
+				</div>
+
+				<div class="footer-contacts">
+					<div class="footer-block block-1">
+						<h5>New York</h5>
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/index.html]: Done.

File: hw1/index.html [hunk 8]
```diff
@@ -0,0 +1,325 @@
...
+
+				  <a class="next" onclick="plusSlides(1)">
+				  	<img src="assets/img/arrow-right.png" alt="arrow right">
+				  </a>	
+				  <a class="prev" onclick="plusSlides(-1)">
+				  	<img src="assets/img/arrow-left.png" alt="arrow left">
+				  </a>
+
+				  <div class="slide-item-content">
+				  	<h3 class="slide-item-header">Botnet</h3>
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/index.html]: Done.

File: hw1/assets/js/stickyNav.js [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+    tabNav.classList.remove("sticky");
+    // main.style.paddingTop = '50px';
+  }
+}
+
+tabLinks.forEach(elem => {
+	elem.onclick = function() {
+		removeActiveClass(tabLinks);
+		elem.classList.add('active');
+	}
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/js/stickyNav.js:41]: Thank's, I will try to fix this bug.

File: hw1/index.html [hunk 10]
```diff
@@ -0,0 +1,325 @@
...
+	<link rel="stylesheet" href="assets/css/main.css">
+</head>
+<body>
+	
+	<header class="main-header">
+		<div class="container main-header__container">
+			<div class="main-header__logo"></div>
+			<div class="main-header__nav">
+				<div class="hamburger-menu">
+				  <input id="menu__toggle" type="checkbox" />
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/index.html:16]: Done.

File: hw1/assets/js/stickyNav.js [hunk 11]
```diff
@@ -0,0 +1,49 @@
...
+	if (pageYOffset >= 30) {
+		headerNav.classList.add('sticky');
+	} else {
+		headerNav.classList.remove('sticky');
+	}
+})
+
+
+function scrollFunc(sticky) {
+	if (window.pageYOffset >= sticky - 65) {
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/js/stickyNav.js]: Done.

File: hw1/assets/sass/_elements.sass [hunk 12]
```diff
@@ -0,0 +1,19 @@
...
+	height: 40px
+	border: 2px solid $accentGreen
+	background-color: $accentGreen
+	color: #fff
+	border-radius: 20px
+	font-size: 12px
+	letter-spacing: 2px
+	outline: none
+	
+@mixin btnOutline
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/sass/_elements.sass:14]: Done.

File: hw1/assets/sass/_fonts.sass [hunk 13]
```diff
@@ -0,0 +1,20 @@
+@font-face
+	font-family: 'aclonica'
+	src: url('../fonts/aclonica-regular-webfont.eot')
+	src: url('../fonts/aclonica-regular-webfont.eot?#iefix') format('embedded-opentype'), url('../fonts/aclonica-regular-webfont.woff2') format('woff2'), url('../fonts/aclonica-regular-webfont.woff') format('woff'), url('../fonts/aclonica-regular-webfont.ttf') format('truetype'), url('../fonts/aclonica-regular-webfont.svg#aclonicaregular') format('svg')
+	font-weight: normal
+	font-style: normal
+
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/sass/_fonts.sass:7]: Done.

File: hw1/assets/sass/_vars.sass [hunk 14]
```diff
@@ -0,0 +1,16 @@
+$grayText: #7F7F7F
+$mainTextColor: #191919
+$darkColor: #1A1A1A
+$lightGray: #E5E5E5
+$midGray: #B2B2B2
+$footerBg: #F2F2F2
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/sass/_vars.sass:6]: Done.

File: hw1/index.html [hunk 15]
```diff
@@ -0,0 +1,325 @@
...
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Skout</title>
+	<link rel="stylesheet" href="assets/css/main.css">
+</head>
+<body>
+	
+	<header class="main-header">
+		<div class="container main-header__container">
+			<div class="main-header__logo"></div>
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/index.html:13]: Done.

File: hw1/index.html [hunk 16]
```diff
@@ -0,0 +1,325 @@
...
+			<div class="main-header__nav">
+				<div class="hamburger-menu">
+				  <input id="menu__toggle" type="checkbox" />
+				  <label class="menu__btn" for="menu__toggle">
+				    <span></span>
+				  </label>
+				  <div class="menu__box">
+				  	<nav class="menu__nav">
+					    <a class="menu__item" href="#">WHY SKOUT</a>
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/index.html:22]: Done.

File: hw1/index.html [hunk 17]
```diff
@@ -0,0 +1,325 @@
...
+	</header>
+
+	<section class="hero">
+		<div class="container">
+			<h1 class="hero__slogan">
+				Tips & Tools
+			</h1>
+			<div class="hero__tabs">
+				<nav>
+					<a href="#tips" class="tab-item">tips</a>
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/index.html:60]: Done

File: hw1/index.html [hunk 18]
```diff
@@ -0,0 +1,325 @@
...
+
+	<main class="main">
+		<section class="tips" id="tips">
+			<div class="container">
+				<div class="section-header">
+					<img src="assets/img/skout-tips.svg" alt="tips">
+				</div>
+				<div class="tips__accordion accordion-block">
+					
+					<button class="accordion acc-1"><span class="accordion-header">System Patching</span> <span class="accordion-icon"></span></button>
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/index.html:76]: Thank's, I will fix it.

File: hw1/index.html [hunk 19]
```diff
@@ -0,0 +1,325 @@
...
+			<div class="glossary__slider">	
+				<div class="container slideshow-container">	
+
+					<!-- Slides content goes here from slider.js -->
+
+				  <a class="next" onclick="plusSlides(1)">
+				  	<img src="assets/img/arrow-right.png" alt="arrow right">
+				  </a>	
+				  <a class="prev" onclick="plusSlides(-1)">
+				  	<img src="assets/img/arrow-left.png" alt="arrow left">
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/index.html:166]: Done

File: hw1/index.html [hunk 20]
```diff
@@ -0,0 +1,325 @@
...
+		</section>
+		<section class="faqs" id="faqs">
+			<div class="container">
+				<div class="section-header">
+					<img src="assets/img/skout-faqs.svg" alt="tips">
+				</div>
+
+				<div class="faqs__accordion accordion-block">
+					
+					<button class="accordion"><span class="accordion-header">What are your system requirements?</span>  <span class="accordion-icon"></span></button>
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/index.html:188]: Thank's, I will fix it.

File: hw1/index.html [hunk 21]
```diff
@@ -0,0 +1,325 @@
...
+
+					<button class="accordion"><span class="accordion-header">Can I get a Demo?</span>  <span class="accordion-icon"></span></button>
+					<div class="accordion__content">
+					  <p>
+					  	Everyone needs cybersecurity. Some information requires more protection than other information, but our solution is adaptable to every business. 
+					  </p>
+					</div>
+
+					<button class="more-faqs-btn">
+						MORE QUESTIONS
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/index.html:227]: done

File: hw1/index.html [hunk 22]
```diff
@@ -0,0 +1,325 @@
...
+			</div>
+		</section>
+	</main>
+
+	<footer class="main-footer">
+		<div class="container">
+			<div class="footer-blocks">
+				<div class="footer-block footer-links">
+					<nav>
+						<a href="#">Blog</a>
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/index.html:241]: Thank;s for all comments, I will fix it as soon as I can)

File: hw1/index.html [hunk 23]
```diff
@@ -0,0 +1,325 @@
...
+						<a href="#">Company</a>
+						<a href="#">Terms & Conditions</a>
+						<a href="#">Privacy Poacy</a>
+						<a href="#">Careers</a>
+					</nav>
+					<p class="main-phone">
+						Call the SOC <br>
+						<span class="phone-number">1-855-838-4500</span>
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/index.html:249]: Thank;s for all comments, I will fix it as soon as I can)

File: hw1/index.html [hunk 24]
```diff
@@ -0,0 +1,325 @@
...
+					</p>
+				</div>
+
+				<div class="footer-contacts">
+					<div class="footer-block block-1">
+						<h5>New York</h5>
+						<p>
+							<span class="phone-number">+1 631-203-6600</span>
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/index.html:257]: Thank;s for all comments, I will fix it as soon as I can)

File: hw1/index.html [hunk 25]
```diff
@@ -0,0 +1,325 @@
...
+						</p>
+						<p class="address">
+							423 W 55th St. 7th Floor <br>
+							New York, NY, 10019
+						</p>
+						<a href="#" class="view-map">View map</a>
+					</div>
+					<div class="footer-block block-2">
+						<h5>Corporate <br>Headquarters</h5>
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/index.html]: Thank;s for all comments, I will fix it as soon as I can)

File: hw1/index.html [hunk 26]
```diff
@@ -0,0 +1,325 @@
...
+			<div class="footer-socials">
+				<ul>
+					<li>
+						<a href="#">
+							<img src="assets/img/twitter.png" alt="Social Link">
+						</a>
+					</li>
+					<li>
+						<a href="#">
+							<img src="assets/img/facebook.png" alt="Social Link">
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/index.html:300]: Thank;s for all comments, I will fix it as soon as I can)

File: hw1/index.html [hunk 27]
```diff
@@ -0,0 +1,325 @@
...
+						</a>
+					</li>
+					<li>
+						<a href="#">
+							<img src="assets/img/linkedin.png" alt="Social Link">
+						</a>
+					</li>
+				</ul>
+				<div class="footer-logo">
+					<img src="assets/img/logo-full.png" alt="Logo">
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/index.html:315]: Thank;s for all comments, I will fix it as soon as I can)

File: hw1/assets/sass/main.sass [hunk 28]
```diff
@@ -0,0 +1,1050 @@
+@import "fonts"
+@import "vars"
+@import "elements"
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/sass/main.sass:3]: Thank;s for all comments, I will fix it as soon as I can)

File: hw1/assets/sass/main.sass [hunk 29]
```diff
@@ -0,0 +1,1050 @@
...
+	z-index: 10
+	top: 0
+	width: 100%
+	padding: 19px 0
+
+	.desktop-menu
+		display: none
+
+	&.sticky
+		background-color: #fff
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/sass/main.sass:49]: Thank;s for all comments, I will fix it as soon as I can)

File: hw1/assets/sass/main.sass (hunk 6 above)
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/sass/main.sass]: Thank;s for all comments, I will fix it as soon as I can)

File: hw1/index.html (hunk 7 above)
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/index.html]: Thank;s for all comments, I will fix it as soon as I can)

File: hw1/assets/sass/main.sass [hunk 30]
```diff
@@ -0,0 +1,1050 @@
...
+			flex-direction: column
+			justify-content: space-between
+			position: fixed
+			visibility: hidden
+			top: 65px
+			right: -100%
+			width: 100%
+			height: calc(100% - 65px)
+			margin: 0
+			padding: 40px 0
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/sass/main.sass:134]: Thank;s for all comments, I will fix it as soon as I can)

File: hw1/assets/sass/main.sass [hunk 31]
```diff
@@ -0,0 +1,1050 @@
...
+						color: #fff
+						background-color: $accentGreen
+
+				.get-skout-btn
+					@include btnPaddingX(54px)
+
+				.log-in-btn
+					@include btnPaddingX(22.5px)
+
+		#menu__toggle:checked ~ .menu__btn > span
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/sass/main.sass:179]: Thank;s for all comments, I will fix it as soon as I can)

File: hw1/assets/sass/main.sass [hunk 32]
```diff
@@ -0,0 +1,1050 @@
...
+			width: 100%
+			text-align: left
+			outline: none
+			transition: 0.4s
+			padding: 15px 0
+			padding-left: 60px
+			transition: $transitionTime
+
+			&:hover
+				background-color: $lightBlue
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/sass/main.sass:309]: Thank;s for all comments, I will fix it as soon as I can)

File: hw1/assets/sass/main.sass [hunk 33]
```diff
@@ -0,0 +1,1050 @@
...
+			top: 0
+			transform: rotate(90deg)
+
+		#menu__toggle:checked ~ .menu__box
+			visibility: visible
+			right: 0
+
+
+	&__logo
+		display: flex
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/sass/main.sass:196]: Thank;s for all comments, I will fix it as soon as I can)

File: hw1/index.html [hunk 34]
```diff
@@ -0,0 +1,325 @@
...
+						  	Malware often takes the form of code, innocuous content, scripts or software. With SKOUT’s End-to-End Protection you can rest assured that all vulnerable points within your system are accounted for and safeguarded so that you don’t fall victim to malware of any kind.
+						 	</p>
+						 	<p class="accordion__footer">
+						  	Based on the 2018 Verizon Data Breach Report
+						  </p>
+					  </div>
+					  
+					</div>
+
+					<button class="accordion acc-3"><span class="accordion-header">Empowering Teams</span> <span class="accordion-icon"></span></button>
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/index.html:113]: Thank's for all comments, I will fix it as soon as I can)

File: hw1/assets/sass/main.sass [hunk 35]
```diff
@@ -0,0 +1,1050 @@
...
+				position: absolute
+				left: 15px
+				top: 0
+				font-family: actor, sans-serif
+				font-size: 40px
+				line-height: 48px
+				letter-spacing: 2px
+				color: $lightGray
+
+			&.acc-1::before
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/sass/main.sass:327]: Thank's for all comments, I will fix it as soon as I can)

File: hw1/assets/sass/main.sass (hunk 1 above)
Comment (VladyslavPolushyn) [reply hw1/assets/sass/main.sass:419]: Thank's for all comments, I will fix it as soon as I can)

File: hw1/assets/sass/main.sass [hunk 36]
```diff
@@ -0,0 +1,1050 @@
...
+				font-size: 42px
+				line-height: 54px
+
+			.accordion:last-of-type.active
+				border-bottom: none
+
+
+
+			span.accordion-header
+					max-width: 360px
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/sass/main.sass:870]: Thank's for all comments, I will fix it as soon as I can)

File: hw1/assets/sass/main.sass [hunk 37]
```diff
@@ -0,0 +1,1050 @@
...
+
+			.accordion__content
+
+				.accordion__text
+					max-width: 365px
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/sass/main.sass:875]: Thank's for all comments, I will fix it as soon as I can)

File: hw1/assets/sass/main.sass [hunk 38]
```diff
@@ -0,0 +1,1050 @@
...
+@media screen and (min-width: 768px)
+
+	.hero
+
+		.container
+
+		&__tabs
+
+			&.sticky
+				left: calc(50% - 770px / 2)
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/sass/main.sass:731]: Thank's for all comments, I will fix it as soon as I can)

File: hw1/assets/sass/main.sass [hunk 39]
```diff
@@ -0,0 +1,1050 @@
...
+@media screen and (min-width: 992px)
+
+	.main-header
+
+		&__logo
+			display: flex
+			align-items: center
+
+			img
+				height: 25px
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/sass/main.sass:742]: No)
Thank's for all comments, I will fix it as soon as I can)

File: hw1/assets/sass/main.sass [hunk 40]
```diff
@@ -0,0 +1,1050 @@
...
+						border-color: $accentGreen
+
+					&:first-child
+						margin-right: 20px
+
+				.get-skout-btn
+					@include btnPaddingX(54px)
+
+				.log-in-btn
+					@include btnPaddingX(22.5px)
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/sass/main.sass:792]: Okay, I will remember this rule
Thank's for all comments, I will fix it as soon as I can)

File: hw1/assets/sass/main.sass [hunk 41]
```diff
@@ -0,0 +1,1050 @@
...
+		&__slogan
+			padding-top: 200px
+			padding-bottom: 100px
+			font-family: actor, sans-serif
+			font-size: 110px
+			line-height: 130px
+
+		&__tabs
+			height: 80px
+			bottom: -36px
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/sass/main.sass:807]: Thank's for all comments, I will fix it as soon as I can)

File: hw1/assets/sass/main.sass [hunk 42]
```diff
@@ -0,0 +1,1050 @@
...
+					content: ''
+					position: absolute
+					left: 0
+					width: 30px
+					height: 30px
+					background: url('../img/i-icon.png') center/cover
+
+	.glossary
+
+		&__slider
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/sass/main.sass:420]: Thank's for all comments, I will fix it as soon as I can)

File: hw1/assets/sass/main.sass [hunk 43]
```diff
@@ -0,0 +1,1050 @@
...
+					opacity: 1
+
+			@keyframes fade 
+				from
+					opacity: .4
+				to
+					opacity: 1
+
+	.faqs
+		padding-top: 30px
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/sass/main.sass:544]: Thank's for all comments, I will fix it as soon as I can)

File: hw1/assets/sass/main.sass [hunk 44]
```diff
@@ -0,0 +1,1050 @@
...
+				line-height: 20px
+				letter-spacing: 0px
+
+				&.main-phone
+					font-family: acme, sans-serif
+					line-height: 24px
+					letter-spacing: 2px
+					text-transform: uppercase
+
+			.phone-number
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/sass/main.sass:662]: Thank's for all comments, I will fix it as soon as I can)

File: hw1/assets/sass/main.sass [hunk 45]
```diff
@@ -0,0 +1,1050 @@
...
+
+					&::before, &::after
+						content: none
+
+				&.active
+					font-size: 16px
+					padding-left: 60px
+
+					span.accordion-icon
+						background: url('../img/arrow-up.png') center/cover
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/sass/main.sass:576]: Thank's for all comments, I will fix it as soon as I can)

[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/sass/main.sass:589]: Done.

File: hw1/assets/sass/main.sass (hunk 43 above)
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/sass/main.sass:557]: Done.

File: hw1/assets/sass/main.sass (hunk 41 above)
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/sass/main.sass:824]: I removed the bottom rule.
Done

File: hw1/assets/sass/main.sass (hunk 39 above)
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/sass/main.sass:756]: Done.

File: hw1/assets/sass/main.sass (hunk 38 above)
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/sass/main.sass:745]: Done

File: hw1/assets/sass/main.sass (hunk 37 above)
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/sass/main.sass:894]: Done

File: hw1/assets/sass/main.sass (hunk 1 above)
Comment (VladyslavPolushyn) [reply hw1/assets/sass/main.sass:431]: Done.

File: hw1/assets/sass/main.sass (hunk 35 above)
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/sass/main.sass:339]: What do you mean?

File: hw1/index.html (hunk 34 above)
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/index.html:119]: Done

File: hw1/assets/sass/main.sass (hunk 35 above)
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/sass/main.sass:339]: Done

File: hw1/assets/sass/main.sass (hunk 33 above)
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/sass/main.sass:200]: I have not heard of such a rule before, thanks.
I hope I'll quickly retrain, thanks.

File: hw1/assets/sass/main.sass (hunk 32 above)
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/sass/main.sass]: Done, thanks

File: hw1/assets/sass/main.sass (hunk 31 above)
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/sass/main.sass:183]: Done.

File: hw1/assets/sass/main.sass (hunk 30 above)
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/sass/main.sass:138]: Thank's, I'll keep it on mind.

File: hw1/assets/sass/main.sass (hunk 29 above)
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/sass/main.sass:52]: Done

File: hw1/index.html (hunk 26 above)
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/index.html:306]: Done

File: hw1/index.html (hunk 25 above)
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/index.html]: Removed <br>

File: hw1/index.html (hunk 23 above)
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/index.html:255]: Done

File: hw1/index.html (hunk 24 above)
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/index.html:263]: Done

File: hw1/index.html [hunk 46]
```diff
@@ -0,0 +1,325 @@
...
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Skout</title>
+	<link rel="stylesheet" href="assets/css/main.css">
+</head>
+<body>
+	
+	<header class="main-header">
```
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/index.html]: Done

File: hw1/index.html (hunk 18 above)
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/index.html:82]: Done

File: hw1/index.html (hunk 20 above)
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/index.html:194]: Done

File: hw1/index.html (hunk 22 above)
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/index.html:247]: Done

File: hw1/assets/sass/main.sass (hunk 44 above)
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/sass/main.sass:676]: Done

File: hw1/assets/js/slider.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/js/slider.js:138]: I will learn the rendering and I hope I can fix it.
But I'm not ready now.

File: hw1/assets/js/stickyNav.js (hunk 9 above)
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/js/stickyNav.js:41]: I will study and try this later. Thank's!

File: hw1/assets/sass/main.sass (hunk 42 above)
[the remark this answers is not in the document]
Comment (VladyslavPolushyn) [reply hw1/assets/sass/main.sass:432]: I will study and try this later. Thank's!

File: hw1/assets/sass/main.sass (hunk 43 above)
[the remark this answers is not in the document]
Comment (marynakorol) [reply hw1/assets/sass/main.sass:547]: image

[the remark this answers is not in the document]
Comment (marynakorol) [reply hw1/assets/sass/main.sass:547]: fix this on future

**Review by marynakorol — APPROVED** [review]

**Pull request merged**